### DISCUSSION TITLE
Keine Verbindung zum WR / HM-600 / Olimex ESP32-POE / gpio_set_level(226): GPIO output gpio_num error

### DISCUSSION THREAD

**Discussion by DelBrai** [tbnobody/OpenDTU#552] [Q&A]
Hallo,

ich würde mich ganz außerordentlich über Hilfre freuen!

Ausgangssituation:

- Olimex ESP32-POE Board mit DollaTek NRF24L01+ sowie Hoymiles HM-600 WR.
- Wechselrichter blinkt grün mit ca. 1Hz -> laut Manual wird Strom produziert.

- Verbindung zwischen Olimex und NRF24L01+ via Flachbandkabel (es kommt der EXT1-Connector auf dem Olimex-Board zum Einsatz).

- Neueste Version von OpenDTU aufs Board geladen.
- OpenDTU verbindet sich per LAN und WLAN gleichermaßen problemlos mit dem Netzwerk.
- OpenDTU meldet eine ordnungsgemäße Verbindung zum Funkchip.

**Einzig: Es wird kein WR gefunden.
Es bestand noch nie eine Verbindung.**

Es sind zwei  DollaTek NRF24L01+ vorhanden, bei einem wurde ein 100uF Elko zwischen GND und VCC verlötet.

Anpassungen in der platform_override.ini

```
[env:olimex_esp32_poe]
; https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware
board = esp32-poe
build_flags = ${env.build_flags}
    -DHOYMILES_PIN_MISO=2
    -DHOYMILES_PIN_MOSI=16
    -DHOYMILES_PIN_SCLK=13
    -DHOYMILES_PIN_IRQ=15
    -DHOYMILES_PIN_CE=36
    -DHOYMILES_PIN_CS=4
    -DOPENDTU_ETHERNET
```

Die Pins sind richtig verbunden, ich habe es dreimal mit dem Multimeter "gepiept".

Die Webansicht der Konsole meldet in Endlosschleife:

```
18:12:14.133 > Fetch inverter: 1141XXXXXXXX
18:12:14.133 > Request SystemConfigPara
18:12:14.259 > TX RealTimeRunData Channel: 23 --> 15 84 51 19 75 80 17 39 04 80 0B 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 40 CB 4F 
18:12:14.526 > RX Period End
18:12:14.526 > All missing
18:12:14.526 > Nothing received, resend whole request
18:12:14.526 > TX RealTimeRunData Channel: 40 --> 15 84 51 19 75 80 17 39 04 80 0B 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 40 CB 4F 
18:12:14.850 > RX Period End
18:12:14.850 > All missing
18:12:14.850 > Nothing received, resend whole request
18:12:14.850 > TX RealTimeRunData Channel: 61 --> 15 84 51 19 75 80 17 39 04 80 0B 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 40 CB 4F 
18:12:15.194 > RX Period End
18:12:15.194 > All missing
18:12:15.194 > Nothing received, resend whole request
18:12:15.194 > TX RealTimeRunData Channel: 75 --> 15 84 51 19 75 80 17 39 04 80 0B 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 40 CB 4F 
18:12:15.523 > RX Period End
18:12:15.523 > All missing
18:12:15.523 > Nothing received, resend whole request
18:12:15.523 > TX RealTimeRunData Channel: 3 --> 15 84 51 19 75 80 17 39 04 80 0B 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 40 CB 4F 
18:12:15.733 > RX Period End
18:12:15.733 > All missing
18:12:15.733 > Nothing received, resend count exeeded
18:12:15.874 > TX AlarmData Channel: 23 --> 15 84 51 19 75 80 17 39 04 80 11 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 9A D0 94 
18:12:16.603 > RX Period End
18:12:16.603 > All missing
18:12:16.603 > Nothing received, resend whole request
18:12:16.603 > TX AlarmData Channel: 40 --> 15 84 51 19 75 80 17 39 04 80 11 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 9A D0 94 
18:12:17.334 > RX Period End
18:12:17.334 > All missing
18:12:17.334 > Nothing received, resend whole request
18:12:17.334 > TX AlarmData Channel: 61 --> 15 84 51 19 75 80 17 39 04 80 11 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 9A D0 94 
18:12:18.056 > RX Period End
18:12:18.056 > All missing
18:12:18.056 > Nothing received, resend whole request
18:12:18.056 > TX AlarmData Channel: 75 --> 15 84 51 19 75 80 17 39 04 80 11 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 9A D0 94 
18:12:18.808 > RX Period End
18:12:18.808 > All missing
18:12:18.808 > Nothing received, resend whole request
18:12:18.808 > TX AlarmData Channel: 3 --> 15 84 51 19 75 80 17 39 04 80 11 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 9A D0 94 
18:12:19.448 > RX Period End
18:12:19.448 > All missing
18:12:19.448 > Nothing received, resend count exeeded
18:12:19.532 > TX SystemConfigPara Channel: 23 --> 15 84 51 19 75 80 17 39 04 80 05 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 8E C4 80 
18:12:19.868 > RX Period End
18:12:19.868 > All missing
18:12:19.868 > Nothing received, resend whole request
18:12:19.868 > TX SystemConfigPara Channel: 40 --> 15 84 51 19 75 80 17 39 04 80 05 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 8E C4 80 
18:12:20.188 > RX Period End
18:12:20.188 > All missing
18:12:20.188 > Nothing received, resend whole request
18:12:20.188 > TX SystemConfigPara Channel: 61 --> 15 84 51 19 75 80 17 39 04 80 05 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 8E C4 80 
18:12:20.556 > RX Period End
18:12:20.556 > All missing
18:12:20.556 > Nothing received, resend whole request
18:12:20.556 > TX SystemConfigPara Channel: 75 --> 15 84 51 19 75 80 17 39 04 80 05 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 8E C4 80 
18:12:20.908 > RX Period End
18:12:20.908 > All missing
18:12:20.908 > Nothing received, resend whole request
18:12:20.908 > TX SystemConfigPara Channel: 3 --> 15 84 51 19 75 80 17 39 04 80 05 00 63 DA 9D 6D 00 00 00 00 00 00 00 00 8E C4 80 
18:12:21.093 > RX Period End
18:12:21.093 > All missing
18:12:21.093 > Nothing received, resend count exeeded

```
Schaut man sich via Putty den seriellen Output an, so wird man erschlagen von einer hohen Zahl von Fehlermeldungen:

```
Nothing received, resend whole request
E (59786) gpio: gpio_set_level(226): GPIO output gpio_num error
TX AlarmData Channel: 61 --> 15 84 51 19 75 80 17 39 04 80 11 00 63 DA 96 7F 00 00 00 00 00 00 00 00 DF 75 6D 
E (59800) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59897) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59898) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59899) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59903) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59909) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59915) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59920) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59926) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59932) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59937) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59943) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59948) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59954) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59960) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59966) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59971) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59977) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59982) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59988) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59994) gpio: gpio_set_level(226): GPIO output gpio_num error
E (59999) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60005) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60011) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60016) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60022) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60027) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60033) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60039) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60045) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60050) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60056) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60061) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60067) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60073) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60078) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60084) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60090) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60095) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60101) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60106) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60112) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60118) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60124) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60129) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60135) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60140) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60146) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60151) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60157) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60163) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60169) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60174) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60180) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60185) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60191) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60197) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60203) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60208) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60214) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60219) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60225) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60230) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60236) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60242) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60248) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60253) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60259) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60264) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60270) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60276) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60282) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60287) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60293) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60298) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60304) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60309) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60315) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60321) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60327) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60332) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60338) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60343) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60349) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60355) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60361) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60366) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60372) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60377) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60383) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60388) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60394) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60400) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60406) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60411) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60417) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60422) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60428) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60434) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60440) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60445) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60451) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60456) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60462) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60467) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60473) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60479) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60485) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60490) gpio: gpio_set_level(226): GPIO output gpio_num error
E (60496) gpio: gpio_set_level(226): GPIO output gpio_num error
RX Period End
All missing
Nothing received, resend whole request
E (60513) gpio: gpio_set_level(226): GPIO output gpio_num error
TX AlarmData Channel: 75 --> 15 84 51 19 75 80 17 39 04 80 11 00 63 DA 96 7F 00 00 00 00 00 00 00 00 DF 75 6D
```

Dieser Fehler treten in hoher Zahl auf, das Putty Log ist in Sekunden auf mehrere KB abgeschwollen.

**Ich habe nun keine Ahnung, wie ich weiter machen soll.
Danke im Voraus für Input aus der Community! :)**

**Comment by MaxDaNr1** [comment]
Ich habe das gleiche Problem. Ich habe zwar das Standard-Set-Up aus der Anleitung, aber irgendwie kommen auch bei mir keine Daten vom WR. (Hier ist mein ursprünglicher Thread: https://github.com/tbnobody/OpenDTU/discussions/307).
Mittlerweile werden die Tage ja wieder länger und ich habe mehr zu rumzuprobieren.
Ich habe verschiedene Netzteile, verschiedene Kabel, verschiedene Entfernungen zum WR und verschiedene Pegel.... alles bisher ohne Erfolg.

**Comment by DelBrai** [comment]
Ich bin weitergekommen. Ich habe die Verdrahtung auf Default geändert, also so, wie in der originalen platform.ini
Das heißt, die GPIOs 4 und 36 sind nun nicht mehr in benutzung, dafür aber 14 und 5.
Das Ganze läuft nun wunderbar. (Was für ein tolles Projekt!)

Aus technischem Interesse würde mich aber schon interessieren, was an GPIO4 und GPIO36 anders ist / warum diese Ausgänge offenbar nicht geeignet sind.

Ich würde mich über Input freuen!

@MaxDaNr1 Ich vermute, dein Problem liegt an anderer Stelle, da du keine "GPIO output gpio_num error" Fehler in Putty siehst. Bist du denn inzwischen weiter gekommen?

**Reply by MaxDaNr1** [reply]
Ich habe nochmal komplett neu compiled mit der neuesten Version. Leider werden immernoch keine Daten empfangen. Ich bin kurz davor einen zweiten WR zu bestellen und mal zu prüfen, ob es vielleicht am WR liegt. Oder vielleicht probiere ich es mal mit Ahoy.

**Comment by MaxDaNr1** [comment]
Ich habe jetzt mal Ahoy ausprobiert. Auch hier werden keine Daten empfangen:

ODE
                SSID: AHOY-DTU
PWD: esp_8266
IP Address: http://192.168.4.1
---------

Wifi event: 0
Wifi event: 2
Wifi event: 10
connect to network 'MyWLAN' ...
I: added inverter 116174404070
reconnect in 9 seconds
Wifi event: 4

[WiFi] Connected
[WiFi] AP disabled
Wifi event: 11
reconnect in 18 seconds
reconnect in 17 seconds
Wifi event: 7


--------------------------------
Welcome to AHOY!

point your browser to http://192.168.2.128 (Station)
to configure your device
--------------------------------

[  3837][E][WiFiGeneric.cpp:1476] hostByName(): DNS Failed for
I: MQTT disconnected, reason: TCP disconnect
I: [NTP]: 2023-02-15 08:43:07 UTC
I: resetPayload: id: 0
I: (#0) enqueuedCmd: 1
I: (#0) enqueuedCmd: 11
I: (#0) enqueuedCmd: 5
I: (#0) sendTimePacket
I: sendTimePacket 1
I: TX 27B Ch40 | 15 74 40 40 70 82 07 13 04 80 01 00 63 EC 9B 32 00 00 00 00 00 00 00 00 2E 7D 77
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 1
I: TX 27B Ch61 | 15 74 40 40 70 82 07 13 04 80 01 00 63 EC 9B 32 00 00 00 00 00 00 00 00 2E 7D 77
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 1
I: TX 27B Ch75 | 15 74 40 40 70 82 07 13 04 80 01 00 63 EC 9B 32 00 00 00 00 00 00 00 00 2E 7D 77
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 1
I: TX 27B Ch3 | 15 74 40 40 70 82 07 13 04 80 01 00 63 EC 9B 32 00 00 00 00 00 00 00 00 2E 7D 77
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 1
I: TX 27B Ch23 | 15 74 40 40 70 82 07 13 04 80 01 00 63 EC 9B 32 00 00 00 00 00 00 00 00 2E 7D 77
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 1
I: TX 27B Ch40 | 15 74 40 40 70 82 07 13 04 80 01 00 63 EC 9B 32 00 00 00 00 00 00 00 00 2E 7D 77
I: resetPayload: id: 0
I: (#0) sendTimePacket
I: sendTimePacket b
I: TX 27B Ch61 | 15 74 40 40 70 82 07 13 04 80 0B 00 63 EC 9B 50 00 00 00 00 00 00 00 00 46 C6 CC
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket b
I: TX 27B Ch75 | 15 74 40 40 70 82 07 13 04 80 0B 00 63 EC 9B 50 00 00 00 00 00 00 00 00 46 C6 CC
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket b
I: TX 27B Ch3 | 15 74 40 40 70 82 07 13 04 80 0B 00 63 EC 9B 50 00 00 00 00 00 00 00 00 46 C6 CC
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket b
I: TX 27B Ch23 | 15 74 40 40 70 82 07 13 04 80 0B 00 63 EC 9B 50 00 00 00 00 00 00 00 00 46 C6 CC
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket b
I: TX 27B Ch40 | 15 74 40 40 70 82 07 13 04 80 0B 00 63 EC 9B 50 00 00 00 00 00 00 00 00 46 C6 CC
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket b
I: TX 27B Ch61 | 15 74 40 40 70 82 07 13 04 80 0B 00 63 EC 9B 50 00 00 00 00 00 00 00 00 46 C6 CC
I: resetPayload: id: 0
I: (#0) sendTimePacket
I: sendTimePacket 5
I: TX 27B Ch75 | 15 74 40 40 70 82 07 13 04 80 05 00 63 EC 9B 6E 00 00 00 00 00 00 00 00 E9 D1 44
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 5
I: TX 27B Ch3 | 15 74 40 40 70 82 07 13 04 80 05 00 63 EC 9B 6E 00 00 00 00 00 00 00 00 E9 D1 44
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 5
I: TX 27B Ch23 | 15 74 40 40 70 82 07 13 04 80 05 00 63 EC 9B 6E 00 00 00 00 00 00 00 00 E9 D1 44
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 5
I: TX 27B Ch40 | 15 74 40 40 70 82 07 13 04 80 05 00 63 EC 9B 6E 00 00 00 00 00 00 00 00 E9 D1 44
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 5
I: TX 27B Ch61 | 15 74 40 40 70 82 07 13 04 80 05 00 63 EC 9B 6E 00 00 00 00 00 00 00 00 E9 D1 44
W: while retrieving data: last frame missing: Request Retransmit
I: (#0) sendTimePacket
I: sendTimePacket 5
I: TX 27B Ch75 | 15 74 40 40 70 82 07 13 04 80 05 00 63 EC 9B 6E 00 00 00 00 00 00 00 00 E9 D1 44
I: resetPayload: id: 0
I: (#0) enqueuedCmd: 1
I: (#0) enqueuedCmd: 11
I: (#0) enqueuedCmd: 5
I: (#0) sendTimePacket


Irgendwie glaube ich langsam, dass entweder mein nRF-Modul spinnt oder vielleicht einfach auf meinen WR eine falsche Seriennummer angegeben ist. Ich habe jetzt mal ein neues nRF-Modul bestellt (ist dann aber schon das dritte das ich ausprobieren werde). Ich werde berichten.

**Comment by x90cr** [comment]
Ich glaube das deine Pins so nicht richtig sind.
Ich habe folgendes pin_mapping hochgeladen damit der POE läuft.
Übrigens läuft das auch mit dem Olimex MOD-NRF24LR problemlos, der wird als NRF24L01+ erkannt.

[
    {
        "name": "ESP32-POE",
        "nrf24": {
            "miso": 15,
            "mosi": 2,
            "clk": 14,
            "irq": 13,
            "en": 16,
            "cs": 5
        },
        "eth": {
            "enabled": true,
            "phy_addr": 0,
            "power": 12,
            "mdc": 23,
            "mdio": 18,
            "type": 0,
            "clk_mode": 3
        }
    }
]
